src/views/profile/edit.ejs

```diff
@@ -164,14 +165,14 @@
         <option value="GY">Guyana</option>
         <option value="HT">Haiti</option>
         <option value="HM">Heard Island and McDonald Islands</option>
-        <option value="VA">Holy See (Vatican City State)</option>
+        <option value="VA">Holy See</option>
```

Comment:
Hmm, I kinda liked having the bracket here lol. What do you think?

Reply:
I suppose the bracket could be readded